controller: {
  left_gain: 3,
  right_gain: 3,
  control_freq: 10,
  ticks_per_rev: 360
}